qt/aqt/overview.py

```diff
@@ -63,6 +64,22 @@ def show(self) -> None:
     def refresh(self) -> None:
         self._refresh_needed = False
         self.mw.col.reset()
+
+        def fetch_data(
+            col: Collection,
+        ) -> Tuple[bool, bool]:
+            is_dyn = col.decks.current()["dyn"]
+            have_buried = col.sched.have_buried()
+            return (is_dyn, have_buried)
+
+        QueryOp(
+            parent=self.mw,
+            op=fetch_data,
+            success=self._render,
+        ).run_in_background()
+
+    def _render(self, data: Tuple[bool, bool]) -> None:
```

Comment:
This is not wrong and doesn't need to be changed, but just a note for next time: in recent Pythons we can use tuple[bool, bool] instead, which saves having to import Tuple.